### ISSUE TITLE
[Bug] History page will not fetch rows for "All" or "TV Shows"

### ISSUE THREAD

**Issue by SteakOverflow** [Tautulli/Tautulli#578]
When on the History page, PlexPy will continuously try to fetch rows when the "All" or "TV Shows" tab is selected. "Movies" and "Music" will successfully list rows. This is persistent across software and hardware restarts.  

Git Branch: master
Git Commit Hash:    65429975203f64a3b524a6e33b8584d4c0e20e9e
Configuration File: C:\Users\Administrator\Downloads\plexpy-master\plexpy-master\config.ini
Database File:  C:\Users\Administrator\Downloads\plexpy-master\plexpy-master\plexpy.db
Backup Directory:   C:\Users\Administrator\Downloads\plexpy-master\plexpy-master\backups
Cache Directory:    C:\Users\Administrator\Downloads\plexpy-master\plexpy-master\cache
Log Directory:  C:\Users\Administrator\Downloads\plexpy-master\plexpy-master\logs
Platform:   Windows 2012Server
Python Version: 2.7.10 (default, May 23 2015, 09:40:32) [MSC v.1500 32 bit (Intel)]

Log file is attached.
[plexpy.log.txt](https://github.com/drzoidberg33/plexpy/files/145853/plexpy.log.txt)

**Comment by JonnyWong16** [comment]
First, have you tried a force refresh (Ctrl+F5)?

**Comment by SteakOverflow** [comment]
Yes, I have tried a force refresh with Ctrl+F5. It did not resolve the issue.

The log file appears to have the following line added when I attempt to load the "All" or "TV Shows" tab. Timestamps are consistent with times I attempted to load the page:

24-Feb-2016 21:08:06 - ERROR   :: CP Server Thread-2 : WebUI :: /history : Uncaught TypeError: Illegal invocation. (jquery-2.1.4.min.js:4)

**Comment by JonnyWong16** [comment]
Did this happen immediately after you updated to v1.3.9? or after a while? Can you PM me your `plexpy.db` file on the [forum](https://forums.plex.tv/profile/JonnyWong16)?

**Comment by SteakOverflow** [comment]
I've had this issue for a few days now. I can't remember the exact date it started, but it was at least 2/22/16. I hadn't used PlexPy in a few days time since I saw this issue on 2/22/16, so I guess it could have been anything from 1.3.7 on out. It was definitely not visible in 1.3.6, from 2/03/16. I PMed you the database file you asked for.

After playing around a bit, I noticed that the default will try to load 25 entries. Changing this to load 10 entries through the dropdown on the History page will load those first 10 entries, but no other entries can be loaded. Trying to go to the next page after loading 10 entries will also fail.

**Comment by JonnyWong16** [comment]
Yeah, I suspect that there is a bad entry in your database somewhere.

**Comment by SteakOverflow** [comment]
Any way for me to fix it?

**Comment by JonnyWong16** [comment]
I'm looking at your `.db` file. I'll let you know. Did you happen to notice anything weird lately, maybe something happened during a stream?

**Comment by SteakOverflow** [comment]
Yes, I've had a few playback issues with Bob's Burgers and Batman: The Animated Series, notably on Android clients. Specifically, Bob's Burgers had a few "false starts", where playback would begin but the stream would never load. A PMS restart was needed to fix that one. Thanks for your PM response on the forums.

**Comment by SteakOverflow** [comment]
So with the help of JonnyWong16, I was able to correct the issue. The problem was caused by a database entry with a missing duration. Jonny suggested that I open the plexpy.db with [SQLiteBrowser](http://sqlitebrowser.org) and find and correct the entry. I found the problem episode in the `session_history_metadata` table and added a duration of 1,290,000 ms (~22 min), which was previously blank. Hope that anyone else having this same issue can fix their problem.
